.github/workflows/webtests.yml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  # Allow this workflow to be called from other repositories.
+  workflow_call:
+
+jobs:
+  run-tests:
```

Comment:
```suggestion
  run-tests:
    # The web tests are run with a limited matrix, because they retrieve data
    #  from the internet and therefor run into rate limits with the full matrix.
```